src/server/utils/auth.js

```diff
@@ -157,7 +155,7 @@ module.exports.validate = (allowedScopes, logger, config) => {
 					let validToken;
 
 					// get client
-					let client = await service.getClient(decodedToken.aud);
+					let client = await service.getClient(decodedToken.payload.aud);
```

Comment:
what happens if this errors? should we catch this with the async handler?

`let [err, client] = await handler(service.getClient(...))`